cognite/client/_api/sequences.py

```diff
@@ -332,8 +431,13 @@ def retrieve(
         Returns:
             List of sequence data
         """
-        col, data = self._retrieve_with_columns(start, end, columns, external_id, id)
-        return data
+        utils._auxiliary.assert_exactly_one_of_id_or_external_id(id, external_id)
+        post_obj = self._process_ids(id, external_id, wrap_ids=True)[0]
+        post_obj.update(self._process_columns(columns))
+        post_obj.update({"inclusiveFrom": start, "exclusiveTo": end})
+        seqdata = []
+        column_response = self._fetch_data(post_obj, lambda data: seqdata.extend(data))
+        return SequenceData(id=id, external_id=external_id, rows=seqdata, columns=column_response)
 
     def retrieve_dataframe(
         self, start: int, end: int, columns: List[Union[int, str]] = None, external_id: str = None, id: int = None
```

Comment:
Missing return type hint.

Reply:
this is normal for dataframe since it may not be imported, no?